manifests/t/Toggl/ToggleTrack/8.0.4/Toggl.ToggleTrack.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Toggl.ToggleTrack
+PackageVersion: 8.0.4
+PackageLocale: en-US
+Publisher: Toggl OÜ
+PublisherUrl: https://toggl.com/
+PublisherSupportUrl: https://support.toggl.com/
+PrivacyUrl: https://toggl.com/track/legal/privacy/
+Author: Toggl
+PackageName: Toggl Track
+PackageUrl: https://support.toggl.com/en/articles/6176883-toggl-track-desktop-app-for-windows
+License: Proprietary
+Copyright: Copyright © 2022 Toggl OÜ
+CopyrightUrl: https://toggl.com/track/legal/terms/
+ShortDescription: A native desktop application for the leading time tracking tool Toggl Track
+Description: Toggl Track for Windows is a native Windows application that can be installed on your computer. It works seamlessly with the web version of Toggl Track, syncing data on the fly. The desktop app also works offline; it stores data locally and sends it to the Toggl Track server once you’re back online.\
+Moniker: toggl
```

```suggestion
Moniker: toggl-track
```
The moniker `toggl` conflicts with Toggl.TogglDesktop